### ISSUE TITLE
Filter menus are not accessible by keyboard with accessibility tool turned on

### ISSUE THREAD

**Issue by saychakraborty89** [IgniteUI/ignite-ui#2245]
## Description  
With the accessibility tool NVDA turned on, the options in menus opened from filter buttons in iggrid are not accessible by keyboard. As discussed in the support issue - C-00235200, I am logging this issue.

 * ignite-ui version: 23.2
 * browser: any

## Steps to reproduce  

1. Go to the sample igGrid [demo.](https://jsfiddle.net/georgianastasov/kn7ru52s/2/)
2. Turn on NVDA.
3. Navigate through keyboard to reach filter button, hit enter to activate the menu. Notice that the drop down menus "Clear filter", "Starts with", "Ends with" etc are not reachable using the keyboard.

## Violated Accessibility Guideline
This violates WCAG success criterion 2.1.1 - Keyboard of WCAG 2.0 level A.

## Result  
The drop down menus "Clear filter", "Starts with", "Ends with" etc. cannot be expanded or collapsed using the keyboard. The user cannot reach the options inside of the drop down and focus is not trapped within the first and the last interactive elements within the menu.

## Expected result  
The menu should be accessible with keyboard even with NVDA turned on.

## Attachments  
I am attaching a sample video for better understanding.
[issue.zip](https://github.com/IgniteUI/ignite-ui/files/14221744/issue.zip)

**Comment by ChronosSF** [comment]
@saychakraborty89 , looks like a bug indeed, wonder what NVDA does to block that input from happening. We'll investigate and try to fix this asap.

**Comment by saychakraborty89** [comment]
Thank you. Will await your findings.

**Comment by ChronosSF** [comment]
To be released with the next SR on Sep 23
